### PR TITLE
W-10707541 property name limits

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/_partials/mq-user-props.adoc [hunk 1]
```diff
@@ -2,3 +2,9 @@
 // tag::mquserprops[]
 User property names can contain up to 42 alphanumeric characters (a-z, A-Z, 0-9), periods (.), hyphens (-), and colons (:). They cannot contain spaces or other characters. 
 // end::mquserprops[]
+
+// MQ max size for headers and user property names
+// tag::mqMaxHeaderProps[]
+The maximum allowed size for headers and user property names in messages
```
Comment (GouthamiKondakindi) [inline modules/ROOT/pages/_partials/mq-user-props.adoc]: Is there any chance customers might interpret this as "headers" and "user property names"? Then they might think the entire header including name and value is limited to 180 bytes

Comment (hannanelson) [reply modules/ROOT/pages/_partials/mq-user-props.adoc]: header property attributes?

Comment (GouthamiKondakindi) [reply modules/ROOT/pages/_partials/mq-user-props.adoc]: How about "header names and user property names" ?

File: modules/ROOT/pages/mq-faq.adoc [hunk 2]
```diff
@@ -284,6 +284,11 @@ More than one in-flight message can occur if the processing time between `anypoi
 
 Anypoint MQ provides direct access to the message ID and payload. You can see the message headers using the Chrome browser and its Network Inspector feature.
 
+== What is the maximum size for headers and user property names in messages?
```
Comment (GouthamiKondakindi) [inline modules/ROOT/pages/mq-faq.adoc]: @hannanelson Should we make the same change here as well? 
"What is the maximum size for header names and user property names in messages?"

**Review by GouthamiKondakindi — APPROVED** [review]
LGTM

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**